src/Microsoft.DocAsCode.Build.Common/ModelAttributeHandlers/HandleModelAttributesContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.Common
+{
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+
+    using Microsoft.DocAsCode.Plugins;
+
+    public class HandleModelAttributesContext
+    {
+        public IHostService host { get; set; }
+        public bool SkipMarkup { get; set; }
```

Comment:
Feel SkipMarkup is better. ShouldSkipMarkup is like the controller's property that tells Markup if it should do markup. While inside context SkipMarkup is Markup's property that it knows if it skips markup...